test/runtests.jl

```diff
@@ -10,6 +10,11 @@ using Test
 
 include("dummy.jl")
 
+@testset "MOI.Bridges" begin
```

Comment:
Move it back down :) Locally you can do
```julia
] activate .
julia> include("test/Bridges/Bridges.jl")
```
so that you don't have to modify `runtests.jl`

Reply:
yes I forgot to re-fix it :)